README.md

```diff
@@ -5,4 +5,18 @@ ASP.NET Web Hooks provide support for sending and receiving WebHooks.
 Please see the blog [Introducing Microsoft ASP.NET WebHooks Preview](http://blogs.msdn.com/b/webdev/archive/2015/09/04/introducing-microsoft-asp-net-webhooks-preview.aspx) 
 for an introduction to Microsoft ASP.NET WebHooks.
 
-Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
\ No newline at end of file
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+##Helpful samples and resources
+Following(s) are helpful resources from our contributors: 
+
+###Samples
+- [GitHub](/samples/#)
```

Comment:
My bad! I missed that message. I am pushing updated PR.
Thanks for your support.
On 29/09/2015 12:28 AM, "Eilon Lipton" <notifications@github.com> wrote:

> In README.md
> <https://github.com/aspnet/WebHooks/pull/7#discussion_r40590875>:
>
> > \ No newline at end of file
> > +Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
> > +
> > +##Helpful samples and resources
> > +Following(s) are helpful resources from our contributors:
> > +
> > +###Samples
> > +- [GitHub](/samples/#)
>
> I think I suggested to change this to say just "All samples" (and remove
> the # from the URL).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/WebHooks/pull/7/files#r40590875>.
>